independent-projects/tools/platform-descriptor-api/src/test/java/io/quarkus/dependencies/test/ExtensionSerializerTest.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.dependencies.test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.dependencies.Extension;
+
+class ExtensionSerializerTest {
+
+	@Test
+	void testLegacyLabelsMapToKeywords() {
+		Extension e = new Extension();
+		
+		String[] input = new String[] { "what", "not", "to", "do"};
+		
+		e.setLabels(input);
+		
+		List<String> list = (List<String>) e.getMetadata().get("keywords");
+		
+		assertNotNull(list);
+		
+		assertArrayEquals(input, list.toArray(new String[0]));
```

Comment:
You can use Assertj instead `assertThat(list).containsExactly(input)`

Reply:
Also why an additional line break per statement?

Reply:
I don't have/need assertj anywhere else and this code using labels will die in next update so didn't bother :)

Reply:
btw. not sure why the additional line breaks - fixed it.